test/UnusedTactic.lean

```diff
@@ -22,3 +22,23 @@ set_option linter.unusedTactic true in
 example : True := by
   congr
   constructor
+
+section allowing_more_unused_tactics
+--  test that allowing more unused tactics has the desired effect of silencing the linter
```

Comment:
Perhaps also add a test mentioning one of these tactics *before* the modification?

Reply:
Thanks, I added `done` to the previous `example`!